code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -1028,7 +1028,11 @@ var/list/datum/dna/hivemind_bank = list()
 	if(amount == 0)
 		return
 
-	var/mob/living/carbon/target = M.changeling_sting(amount, /obj/item/verbs/changeling/proc/changeling_chemsting, allow_self = bool=="Yes"?TRUE:FALSE)
+	var/mob/living/carbon/target
```

Comment:
what in tarnation is going on here?

Reply:
Most likely forgot to rebase

Reply:
My code just won't compile unless I break up that ternary operator. Beats me.

Reply:
Wrong button
Also that ternary operator is very stupid
See if you can spot why

Reply:
jesus that ternary